src/main/java/io/confluent/connect/elasticsearch/RetryUtil.java

```diff
@@ -160,6 +160,7 @@ public static long computeRetryWaitTimeInMillis(int retryAttempts,
         );
         return function.call();
       } catch (IOException e) {
+        log.warn("Failed to {} due to {}. Retrying attempt ({}/{})", description, e.getCause(), attempt, maxAttempts);
```

Comment:
First the error log here won't be accurate if the attempt is equal to max attempts. 

I think we can distinguish between a warn message while we keep retrying and a similar error message when we exhaust retries and before we throw (I think exception will be printed but should be fine to have the symmetric error message like [before](https://github.com/confluentinc/kafka-connect-elasticsearch/pull/116/files#diff-b929a6768a74899db0afa5ad8eba4f2a0fafb65c9df9372085dfb01e840c02eeR367))

Reply:
Then I think it might be useful to print the backoff along with the warn message when we retry to know how long (in ms) the task will wait. So, that means that the log should be added in two cases (error and warn) and should be moved further down. Wdyt?